desc/objectives/getters.py

```diff
@@ -191,14 +195,84 @@ def get_NAE_constraints(
         FixAxisZ(eq=desc_eq, normalize=normalize, normalize_target=normalize),
         FixPsi(eq=desc_eq, normalize=normalize, normalize_target=normalize),
     )
-
     if profiles:
         for name, con in _PROFILE_CONSTRAINTS.items():
             if getattr(desc_eq, name) is not None:
                 constraints += (
                     con(eq=desc_eq, normalize=normalize, normalize_target=normalize),
                 )
 
+    constraints += (
+        FixNearAxisR(
+            eq=desc_eq,
+            nae_eq=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+        FixNearAxisZ(
+            eq=desc_eq,
+            nae_eq=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+    )
+    if fix_lambda or (fix_lambda >= 0 and type(fix_lambda) is int):
```

Comment:
Why is `fix_lambda=False` by default? Shouldn't lambda be constrained to get the same NAE behavior?

Reply:
No, we have found that it is a lot of the time not needed to fix lambda, and only at higher order have I seen that fixing lambda on-axis helps get the NAE behavior desired